respokeSDK/src/main/java/com/digium/respokesdk/RestAPI/APITransaction.java

```diff
@@ -56,7 +58,18 @@
     protected String params;
     protected int serverResponseCode;
     private AsyncTransaction asyncTrans;
-    
+
+    public static String getSDKHeader() {
+        String sdkTitle = "Respoke-Android";
+        String sdkVersion = BuildConfig.VERSION_NAME;
+        String androidVersion = android.os.Build.VERSION.RELEASE;
+
+        if (!sdkVersion.equals("")) {
+            sdkTitle = sdkTitle + "/" + sdkVersion;
+        }
+
+        return String.format("%s (Android %s)", sdkTitle, androidVersion);
+    }
```

Comment:
I don't really know if APITransaction class is the best place for this function, but I couldn't come up with a better spot for it.